src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.spec.ts

```diff
@@ -139,63 +137,10 @@ describe('PaypalCommercePaymentStrategy', () => {
         document.body.removeChild(container);
     });
 
-    describe('Country test mode on', () => {
```

Comment:
Why are we removing these tests?

Reply:
I don't know why, but github shows edits, that already in master brunch. https://github.com/bigcommerce/checkout-sdk-js/commit/bf2ff1ad874e7e79bb84ac1fbe395db045354f42
Sorry for that, will try to update PR.

Reply:
@animesh1987 updated PR, now it shows correct changes